src/assets/env-json-config.json

```diff
@@ -1,7 +1,6 @@
 {
     "modules": {
         "core": {
-            "base": "SERVER_HOST/api",
```

Comment:
This is something we should probably clean up, but it is connected with the build process, the eUI app has this predisposition as a REST driven app and this is part of this, defining this api path that should match the server endpoint.